database/migrations/mysql/1583505136_add_vsphere_pke_clusters_table.up.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE `vsphere_pke_clusters` (
```

Comment:
Why isn't there a separate table for node pools? It would make managing node pools easier, especially in case of vsphere where there isn't any kind of concept remotely resembling node pools.

Reply:
created separate table for node pools